projects/js-packages/charts/src/components/pie-semi-circle-chart/pie-semi-circle-chart.tsx

```diff
@@ -1,6 +1,6 @@
 import { localPoint } from '@visx/event';
 import { Group } from '@visx/group';
-import Pie, { type PieArcDatum } from '@visx/shape/lib/shapes/Pie';
```

Comment:
we shouldn't import from `lib` or `esm` because this should be determined by the bundler to either use ESM or CJS.